docs/dev/guidelines/database.rst

```diff
@@ -0,0 +1,236 @@
+
+In JPA lazy fetching can be set on any relationship using the fetch attribute. The fetch can be set to either ``LAZY`` or ``EAGER`` as defined in the ``FetchType`` enum. The default fetch type is ``LAZY`` for all relationships except for **OneToOne** and **ManyToOne**, but in general it is a good idea to make every relationship ``LAZY``. The ``EAGER`` default for **OneToOne** and **ManyToOne** is for implementation reasons (more easier to implement), not because it is a good idea. |br|
+
+We **always** use ``FetchType.LAZY``, unless there is a very strong case to be made for ``FetchType.EAGER``.
+
+       .. note::
+        Additional effort to use ``FetchType.LAZY`` does count as a strong argument.
+
+2. Relationships
+================
+
+A relationship is a reference from one object to another. In a relational database relationships are defined through foreign keys. The source row contains the primary key of the target row to define the relationship (and sometimes the inverse). A query must be performed to read the target objects of the relationship using the foreign key and primary key information. In Java, if a relationship is to a collection of other objects, a ``Collection`` or ``array`` type is used in Java to hold the contents of the relationship. In a relational database, collection relations are either defined by the target objects having a foreign key back to the source object's primary key, or by having an intermediate join table to store the relationship (both objects' primary keys). |br|
+
+In this section, we depict common entity relationships we use in Artemis and show some code snippets. 
+
+* **OneToOne** A unique reference from one object to another. It is also inverse of itself. Example: one ``Complaint`` has a reference to one ``Result``.
+
+ .. code:: java
+
+    // Complaint.java
+    @OneToOne
+    @JoinColumn(unique = true)
+    private Result result;
+
+* **OneToMany** A ``Collection`` or ``Map`` of objects. It is the inverse of a **ManyToOne** relationship. Example: one ``Result`` has a list of ``Feedback`` elements. For ordered OneToMany relations see :ref:`ordered collections <ordered>`.
+
+ .. code:: java
+
+    // Result.java
+    @OneToMany(mappedBy = "result", cascade = CascadeType.ALL, orphanRemoval = true)
+    @OrderColumn
+    @JsonIgnoreProperties(value = "result", allowSetters = true)
+    @Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
+    @JsonView(QuizView.Before.class)
+    private List<Feedback> feedbacks = new ArrayList<>();
+
+
+* **ManyToOne** A reference from one object to another. It is the inverse of a **OneToMany** relationship. Example: one ``Feedback`` has a reference to one ``Result``. 
```

Comment:
[...] inverse of **an** OneToMany relationship. [...]

Reply:
... **a** OneToMany ... is actually the correct form here for the following reason:
The word “one” starts with a vowel, but the sound it makes is the consonant “w.”